server/scripts/postgresImport.js

```diff
@@ -0,0 +1,11 @@
+// IMPORT VIA COMMAND LINE
+// psql -d sdc -f server/scripts/postgresSchema.sql
+
+// Copy data
+// COPY itemReviews(itemId,id,name,stars,date,country,review,image,title,avatar,foundThisHelpful) FROM '/Users/leilei/Local/GitProjects/HRR45-SDC-DATAONLY/10MReviews.csv' WITH DELIMITER ',' CSV HEADER;
```

Comment:
How do you handle Postgres flattening all column names without quotation marks into lowercase?

I believe this will produce an error, given that you're keeping them camelcase in the schema below.